Tests/Test/ResponseTest.swift

```diff
@@ -151,10 +151,14 @@ class ResponseTest: XCTestCase {
 
     func testBuildResponseWillAlterFileWithPutBody() {
         let body = "My=Foo"
-        let request = "PUT /form\r\nConnection: Keep-Alive\r\n\r\n\(body)" 
+        let request1 = "POST /form\r\nConnection: Keep-Alive\r\n\r\n\(body)"
+        let parsedRequest1 = parser.parseRequest(request: request1) 
+        let _ = response.buildResponse(request: parsedRequest1)
 
-        let parsedRequest = parser.parseRequest(request: request) 
-        let _ = response.buildResponse(request: parsedRequest)
+        let request2 = "PUT /form\r\nConnection: Keep-Alive\r\n\r\n\(body)" 
```

Comment:
Of course this test will pass; each request uses the same value for `body`.

Reply:
Amended a change here.